README.md

```diff
@@ -3,16 +3,27 @@ Example CNF
 
 [![PR Validation](https://github.com/openshift-kni/example-cnf/actions/workflows/push.yaml/badge.svg)](https://github.com/openshift-kni/example-cnf/actions/workflows/push.yaml)
 
-Example CNF is an OpenShift workload to exercice an SRIOV setup.
+Example CNF is an OpenShift workload to exercice an SRIOV setup, such as the following:
 
 ![Schema](documentation/schema.png)
 
 It is providing the following operators:
 
-* testpmd-operator
 * trex-operator
+    * It provides TRex Traffic Generator, decomposed in two components: a TRex server, deployed as `trexconfig-<x>` pod, which takes care of deploying and configuring a TRex server instance, and a TRex application, deplayed as `trex-app` job, that starts TRex server, generating traffic towards the system under test.
+    * In `trexconfig-<x>` logs, you can see the trex statistics printed periodically. The summary of the test execution can be seen at the end of the `trex-app` job logs.
+    * In our case, as we will see later on, the `trexconfig-<x>` pod has two interfaces connected to the same SRIOV network.
```

Comment:
we could keep docs a bit more factual by omiting "In our case, as we will see later on,"